operator/config/crd/bases/forklift.konveyor.io_plans.yaml

```
@@ -141,6 +141,9 @@ spec:
                 description: Preserve the CPU model and flags the VM runs with in
                   its oVirt cluster.
                 type: boolean
+              preserveStaticIPs:
+                description: Preserve static IPs of VMs in vSphere cluster
```

```suggestion
                description: Preserve static IPs of VMs in vSphere
```
and how about setting the expectation properly by saying it applies to Windows guests only?

---

added, please re-check

---

looks good now